From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Peter Jones <pjones@redhat.com>
Date: Fri, 11 Mar 2022 12:47:20 -0500
Subject: [PATCH] .gitignore: add compile_commands.json and .cache/

These are used by bear/cnc/clangd/etc, but there's no reason to trip
over them all the time.

Signed-off-by: Peter Jones <pjones@redhat.com>
---
 .gitignore | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.gitignore b/.gitignore
index bf0617b..7425432 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,3 +1,4 @@
+.cache/
 .*.d
 .*.P
 .*.sw?
@@ -26,6 +27,7 @@
 /*.rpm
 *-8be4df61-93ca-11d2-aa0d-00e098032b8c
 *-d719b2cb-3d3a-4596-a3bc-dad00e67656f
+compile_commands.json
 core.*
 cov-int/
 pwfile
